app/controllers/alaveteli_pro/plans_controller.rb

```diff
@@ -6,6 +6,9 @@ class AlaveteliPro::PlansController < AlaveteliPro::BaseController
   before_action :authenticate, :check_existing_subscription, only: [:show]
 
   def index
+    plan_name = add_stripe_namespace('pro')
+    stripe_plan = Stripe::Plan.retrieve(plan_name)
+    @plan = AlaveteliPro::WithTax.new(stripe_plan)
```

Comment:
Looks like there's a spec missing here?

Reply:
split it into a controller spec that checks the assignment of `@plan` and a new view spec file that checks that the page renders the formatted currency value as expected